libafl/src/stages/sync.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
     executors::{Executor, ExitKind, HasObservers},
     fuzzer::{Evaluator, EvaluatorObservers, ExecutionProcessor},
     inputs::{Input, InputConverter, UsesInput},
-    stages::{RetryRestartHelper, Stage},
+    stages::{HashSet, RetryRestartHelper, Stage},
```

Comment:
Why `stages::HashSet`?

Reply:
In general, I'd just do a vec, it'll be a lot faster for a small number of entries